### PR TITLE
switch to new Makie

### PR DESCRIPTION
**Pull request by SimonDanisch**
...And get rid of some work arounds! :)

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -1,22 +1,22 @@
+authors = ["Beacon Biosignals, Inc."]
 name = "Lighthouse"
 uuid = "ac2c24cd-07f0-4848-96b2-1b82c3ea0e59"
-authors = ["Beacon Biosignals, Inc."]
 version = "0.12.0"
```
Comment (hannahilea) [inline Project.toml:4]: can go ahead and bump patch version if you like

File: test/learn.jl [hunk 2]
```diff
@@ -142,7 +142,7 @@ end
         @testplot confusion_row
 
         confusion_col = plot_confusion_matrix(plot_data["confusion_matrix"],
-                                              plot_data["class_labels"], :Row)
+                                              plot_data["class_labels"], :Column)
```
Comment (hannahilea) [inline test/learn.jl:145]: ...I'm kind of surprised that this didn't break the percy test?

Comment (SimonDanisch) [reply test/learn.jl:145]: Yeah, that was an embarrassing oversight :D I approved the percy build. so now it uses this as the baseline

**Review by hannahilea — APPROVED** [review]
Looks good to me, once you 
- [x] get the documentation build passing and
- [x] bump the version
!

**Pull request merged**